llvm-core/clang-linker-config/clang-linker-config-20.1.8.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Default linker configuration for clang"
+HOMEPAGE="https://clang.llvm.org/"
+S=${WORKDIR}
+
+LICENSE="metapackage"
```

Comment:
Sorry I didn't notice this sooner, but since this package installs files, it's not a `metapackage`. I guess `public-domain` will work for this kind of config.

(Yes, I know we forgot to update `clang-runtime` after moving the configs.)

Reply:
Done